appveyor.yml

```diff
@@ -0,0 +1,22 @@
+environment:
+  matrix:
+    - nodejs_version: "4"
```

Comment:
No but I'm deciding right now that the only node version we support on Windows will be the latest.